webui/react/src/pages/TrialDetails.tsx

```diff
@@ -179,7 +179,7 @@ const TrialDetailsComp: React.FC = () => {
     const metricRenderer = (metricName: MetricName) => {
       const metricCol = (_: string, record: Step) => {
         const value = extractMetricValue(record, metricName);
-        return value ? <HumanReadableFloat num={value} /> : undefined;
+        return value != null ? <HumanReadableFloat num={value} /> : undefined;
```

Comment:
`!=` instead of `!==` on purpose?

Reply:
Ah yes, so `!==` will check against only `null`. `value != null` is the same as `value !== null && value !== undefined`. `value == null` is equivalent to `value === null || value === undefined`.

Basically it protects against both `null` and `undefined`.

Reply:
https://stackoverflow.com/questions/16607761/why-null-undefined-is-true-in-javascript

Reply:
Thanks :)